share/man/man1/brew.1

```diff
@@ -554,10 +554,6 @@ HOMEBREW_BROWSER
 If set, uses this setting as the browser when opening project homepages, instead of the OS default browser\.
 .
 .TP
-HOMEBREW_BUILD_FROM_SOURCE
```

Comment:
> Security comes from many aspects. The code execution problem can be migrated by vm.

We don't provide a VM. I'd be OK with `BUILD_FROM_SOURCE` needing a VM though, but I don't think that's what you're suggesting. :wink:

> What I mean here that building from source is more security than using third party binaries is focusing one of the aspects: there is no perfect way to verify whether a binary file is comprised. Even with reproducible bottles is not perfect.

With reproducible builds: it's as perfect as `sha256` is. If you don't trust our bottle `sha256`: why would you trust our formula files?

> I think it's naive to think everyone have the same need and we should offer an unify UX for all them. What we should do is making the default UX to favor the majority need. At the same time, we should individuals custom their UX based on their own. And this is including (a) keep such feature unless it's too expensive to maintain (b) well document the optional features (which by nature means it would be used in only small group)

Allow customisation of this sort makes software worse. Look through our issue tracker and tell me honestly if all users were using binaries we would have more issues rather than less. I don't mean to pull rank but having worked on Homebrew for a long time now: binary packages have solved more support issues than any other individual feature.